src/chaplin/lib/route.coffee

```diff
@@ -7,9 +7,7 @@ Controller = require 'chaplin/controllers/controller'
 utils = require 'chaplin/lib/utils'
 
 isEmpty = (object) ->
-  for own key, value of object
-    return false
-  true
+  not Object.keys(object).length
```

Comment:
We decided to transit to ES5 and drop IE8 support.
[PR](https://github.com/chaplinjs/chaplin/pull/875)